yoda/handler.go

```diff
@@ -233,12 +233,13 @@ func handleRawRequest(c *Context, l *Logger, req rawRequest, key keyring.Info, i
 	}
 
 	result, err := c.executor.Exec(exec, req.calldata, map[string]interface{}{
-		"BAND_CHAIN_ID":    vmsg.ChainID,
-		"BAND_VALIDATOR":   vmsg.Validator,
-		"BAND_REQUEST_ID":  strconv.Itoa(int(vmsg.RequestID)),
-		"BAND_EXTERNAL_ID": strconv.Itoa(int(vmsg.ExternalID)),
-		"BAND_REPORTER":    hex.EncodeToString(pubkey.Bytes()),
-		"BAND_SIGNATURE":   sig,
+		"BAND_CHAIN_ID":       vmsg.ChainID,
```

Comment:
@RogerKSI Should we add data source id to Verification msg (to make sure reporter mean to get data from data source cannot manipulate data source id later)

Reply:
ok, agree.

Reply:
Finished